content/en/platform/corda/4.12/enterprise/app-upgrade-notes-enterprise.md

```diff
@@ -83,7 +83,7 @@ been removed.
 
    You can find instructions for both options in [Running tests in IntelliJ]({{< relref "testing.md#running-tests-in-intellij" >}}).
 
-4. Check you're using Corda Gradle plugins version 5.0.12, and that Corda Enterprise dependencies are referenced with the right distribution.
+4. Check you're using Corda Gradle plugins version 5.0.18, and that Corda Enterprise dependencies are referenced with the right distribution.
```

Comment:
Please also add the changes to this file from this PR:

https://github.com/corda/corda-docs-portal/pull/2176/files#diff-f841687c1077226686eda2af744b4474dbd9f8eed7a748eb8ae27b6223859edc